templates/serverpod_templates/projectname_server_upgrade/deploy/aws/scripts/install_dependencies

```diff
@@ -7,7 +29,7 @@ After=multi-user.target
 [Service]
 User=ec2-user
 WorkingDirectory=/home/ec2-user
-ExecStart=/home/ec2-user/serverpod/active/projectname_server/deploy/aws/scripts/run_serverpod
```

Comment:
Suggestion: Revert this name, the string `projectname_server` is significant in that we have other scripts that will replace this string with the actual project name.